src/Quadmath.jl

```diff
@@ -107,9 +106,7 @@ reinterpret(::Type{Float128}, x::Int128) =
     reinterpret(Float128, reinterpret(UInt128, x))
 
 function __init__()
```

Comment:
remove `__init__` block, and move the `include` statement to the bottom (next to the printf one)